packages/marshaller/src/ddl2/activities/filter.ts

```diff
@@ -100,16 +100,27 @@ export class ColumnMapping extends PropertyExt {
 
     remoteValue(filterSelection: any[]) {
         const rf = this.remoteField();
-        let fs = filterSelection.length ? filterSelection[0][rf] : undefined;
-        const isString = typeof fs === "string";
-        if (isString) {
-            fs = fs.trim();
+        let fs = filterSelection.length ? filterSelection.map(sel => {
+            let retVal = sel[rf];
+            const isString = typeof retVal === "string";
+            if (isString) {
+                retVal = retVal.trim();
+            }
+            return retVal;
+        }).join(", ") : undefined;
```

Comment:
The `.join(", ")` will cause everything to return as a string (unless it's undefined)

Reply:
...which will cause issues in the switch statement within `createFilter`

Reply:
Good catch - this is a potential issue:  https://github.com/hpcc-systems/Visualization/issues/3051